unitTests/ui/FacetSearchElementTest.ts

```diff
@@ -0,0 +1,45 @@
+import { FacetSearchElement } from '../../src/ui/Facet/FacetSearchElement';
+import { mock } from '../MockEnvironment';
+import { FacetSearch } from '../../src/ui/Facet/FacetSearch';
+import { $$ } from '../../src/utils/Dom';
+
+export function FacetSearchElementTest() {
+  describe('FacetSearchElement', () => {
+    let facetSearchElement: FacetSearchElement;
+    beforeEach(() => {
+      facetSearchElement = new FacetSearchElement(mock(FacetSearch));
+      facetSearchElement.build();
+    });
+
+    describe('when calling emptyAndShowNoResults', () => {
+      let noResultsElement: HTMLElement;
+      beforeEach(() => {
+        facetSearchElement.emptyAndShowNoResults();
+        [noResultsElement] = $$(facetSearchElement.searchResults).children();
+      });
+
+      it('searchResults contains a single element', () => {
+        expect($$(facetSearchElement.searchResults).children().length).toEqual(1);
+      });
+
+      it('renders a noResults element as a HTMLLIElement', () => {
+        expect(noResultsElement instanceof HTMLLIElement).toBeTruthy();
+      });
+
+      it('renders a noResults element with an id and a class', () => {
+        expect($$(noResultsElement).hasClass('coveo-facet-value-not-found'));
```

Comment:
Should this expectation have a `.toBe(true)` chained to it?

Reply:
Indeed ;)

Reply:
Woops, that's right.